packages/big-design/src/components/Counter/styled.ts

```diff
@@ -0,0 +1,19 @@
+import { remCalc, theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { StyledInput, StyledInputWrapper, StyledInputWrapperProps } from '../Input/private';
+
+export const StyledCounterWrapper = styled(StyledInputWrapper)<StyledInputWrapperProps>`
+  padding-left: ${({ theme }) => theme.spacing.xxSmall};
+  padding-right: ${({ theme }) => theme.spacing.xxSmall};
+  width: ${remCalc(125)};
+  max-width: ${remCalc(320)};
```

Comment:
Having `max-width` and `width` doesn't really make sense. You'll probably just want to pick one or the other.